src/langapi/language_ui.cpp

```diff
@@ -112,8 +113,12 @@ bool language_uit::final()
 {
   language_files.set_message_handler(*message_handler);
 
+  // TODO: This should be moved elsewhere because it is not used in
+  //       this repository.
   // Enable/disable stub generation for opaque methods
-  bool stubs_enabled=_cmdline.isset("generate-opaque-stubs");
+  bool stubs_enabled = false;
+  if(options != nullptr)
+    stubs_enabled = options->is_set("generate-opaque-stubs");
```

Comment:
Is this any easier to read if written as `bool stubs_enabled = options != nullptr ? options->is_set(..) : false;` ?

Reply:
Will be removed in #3520 anyway.